liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -964,6 +964,20 @@ private void installEmptyEAR(MavenProject earProject) throws MojoExecutionExcept
         }
     }
 
+    protected void purgeLocalRepositoryModule(MavenProject project) throws MojoExecutionException {
+        Plugin plugin = getPlugin("org.apache.maven.plugins", "maven-dependency-plugin");
+        String goal = "purge-local-repository";
+        Xpp3Dom config = ExecuteMojoUtil.getPluginGoalConfig(plugin, goal, log);
+        config = Xpp3Dom.mergeXpp3Dom(configuration(
+            element(name("reResolve"), "false"), 
+            element(name("actTransitively"), "false"), 
+            element(name("manualIncludes"), project.getGroupId()+":"+project.getArtifactId())
+            ), config);
+        log.info("Running maven-dependency-plugin:" + goal);
+        log.debug("configuration:\n" + config);
+        executeMojo(plugin, goal(goal), config, executionEnvironment(project, session, pluginManager));
```

Comment:
Along the same lines as above, consider cloning the current session and setting the current project to the project module:
```java
 MavenSession tempSession = session.clone();
 tempSession.setCurrentProject(project);
executeMojo(plugin, goal(goal), config, executionEnvironment(project, tempSession, pluginManager));
```